src/main/java/com/cburch/logisim/analyze/model/Implicant.java

```diff
@@ -266,10 +300,25 @@ public static Expression toExpression(int format, AnalyzerModel model, List<Impl
 
   final int unknowns;
   final int values;
+  final boolean isDontCare;
+  boolean isPrime = true;
```

Comment:
can we have `var` here too?

Reply:
Global variables can not be var'ed unfortunately.